pkg/operator/operator.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func WaitForTektonConfigCR(cs *clients.Clients, rnames utils.ResourceNames) {
-	EnsureTektonConfigExists(cs.TektonConfig(), rnames)
+	EnsureTektonConfigExists(cs.TektonConfig(), rnames) //nolint
```

Comment:
Why don't we want to lint this line?

Reply:
We are not checking the return value here. but this is covered in next line as part of `k8s.ValidateDeployments()`